### PR TITLE
documentation pages updated

### PR DESCRIPTION
**Pull request by shrey27**

### CODE REVIEW

**Review by Sharmaa-Ansh — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          works.
+        </p>
+        <p class="md text--dark xs-s">
+          Apollo-UI is a one stop solutions to all your styling needs. Its
+          modular, clean and integrates smoothly with any framework and library.
+          Go through this documentaion to know more about usage and code
+          structure.
+        </p>
+        <div class="flex-ct-ct">
+          <a href="/installation" class="btn btn--dark btn--md">Docs</a>
```
Comment (OmKakatkar) [inline index.html:58]: The link to installation page is broken. I think the html file is missing in the URL

File: installation/index.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        <div class="box-sm-contact flex-ct-ct slider">
+          <i class="fas fa-bars btn btn--float btn--round lg sb"></i>
+        </div>
+      </header>
+
+      <aside class="box__aside">
+        <section class="box__aside__options">
+          <div class="list--container">
+            <span class="list__title"
+              ><a href="/installation">Installation</a></span
```
Comment (OmKakatkar) [inline installation/index.html:64]: Broken Link, html file missing in URL

File: installation/index.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            >
+          </div>
+          <div class="list--container">
+            <span class="list__title">Utilities</span>
+            <ul class="list">
+              <li><a href="../color">Color Palette</a></li>
```
Comment (OmKakatkar) [inline installation/index.html:70]: Broken Link, html file missing in URL

File: installation/index.html [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+              <li><a href="../typography">Typography</a></li>
```
Comment (OmKakatkar) [inline installation/index.html:71]: Broken Link, html file missing in URL

File: installation/index.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+          <div class="list--container">
+            <span class="list__title">Components</span>
+            <ul class="list">
+              <li><a href="../components/avatar">Avatar</a></li>
+              <li><a href="../components/alert">Alert</a></li>
+              <li><a href="../components/badge">Badge</a></li>
+              <li><a href="../components/button">Button</a></li>
+              <li><a href="../components/card">Card</a></li>
+              <li><a href="../components/image">Image</a></li>
+              <li><a href="../components/input">Input</a></li>
```
Comment (OmKakatkar) [inline installation/index.html:83]: Broken Links, html file missing in URL

File: components/input/index.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+          <div class="list--container">
+            <span class="list__title">Components</span>
+            <ul class="list">
+              <li><a href="../avatar">Avatar</a></li>
+              <li><a href="../alert">Alert</a></li>
+              <li><a href="../badge">Badge</a></li>
+              <li><a href="../button">Button</a></li>
+              <li><a href="../card">Card</a></li>
+              <li><a href="../image">Image</a></li>
+              <li><a href="../input">Input</a></li>
```
Comment (OmKakatkar) [inline components/input/index.html:83]: Broken Links, HTML file missing in URL

File: components/image/index.html [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+          <div class="list--container">
+            <span class="list__title">Components</span>
+            <ul class="list">
+              <li><a href="../avatar">Avatar</a></li>
+              <li><a href="../alert">Alert</a></li>
+              <li><a href="../badge">Badge</a></li>
+              <li><a href="../button">Button</a></li>
+              <li><a href="../card">Card</a></li>
+              <li><a href="../image">Image</a></li>
+              <li><a href="../input">Input</a></li>
```
Comment (OmKakatkar) [inline components/image/index.html:83]: Broken Links, HTML file missing in URL

File: components/button/index.html [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+          <div class="list--container">
+            <span class="list__title">Components</span>
+            <ul class="list">
+              <li><a href="../avatar">Avatar</a></li>
+              <li><a href="../alert">Alert</a></li>
+              <li><a href="../badge">Badge</a></li>
+              <li><a href="../button">Button</a></li>
+              <li><a href="../card">Card</a></li>
+              <li><a href="../image">Image</a></li>
+              <li><a href="../input">Input</a></li>
```
Comment (OmKakatkar) [inline components/button/index.html:83]: Broken Links, HTML file missing in URL

File: components/badge/index.html [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+          <div class="list--container">
+            <span class="list__title">Components</span>
+            <ul class="list">
+              <li><a href="../avatar">Avatar</a></li>
+              <li><a href="../alert">Alert</a></li>
+              <li><a href="../badge">Badge</a></li>
+              <li><a href="../button">Button</a></li>
+              <li><a href="../card">Card</a></li>
+              <li><a href="../image">Image</a></li>
+              <li><a href="../input">Input</a></li>
```
Comment (OmKakatkar) [inline components/badge/index.html:83]: Broken Links, HTML file missing in URL

File: components/avatar/index.html [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+            <div class="list--container">
+              <span class="list__title">Components</span>
+              <ul class="list">
+                <li><a href="../avatar">Avatar</a></li>
+                <li><a href="../alert">Alert</a></li>
+                <li><a href="../badge">Badge</a></li>
+                <li><a href="../button">Button</a></li>
+                <li><a href="../card">Card</a></li>
+                <li><a href="../image">Image</a></li>
+                <li><a href="../input">Input</a></li>
```
Comment (OmKakatkar) [inline components/avatar/index.html:92]: Broken Links, HTML file missing in URL

File: components/alert/index.html [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+          <div class="list--container">
+            <span class="list__title">Components</span>
+            <ul class="list">
+              <li><a href="../avatar">Avatar</a></li>
+              <li><a href="../alert">Alert</a></li>
+              <li><a href="../badge">Badge</a></li>
+              <li><a href="../button">Button</a></li>
+              <li><a href="../card">Card</a></li>
+              <li><a href="../image">Image</a></li>
+              <li><a href="../input">Input</a></li>
```
Comment (OmKakatkar) [inline components/alert/index.html:83]: Broken Links, HTML file missing in URL

File: color/index.html [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+          <div class="list--container">
+            <span class="list__title">Components</span>
+            <ul class="list">
+              <li><a href="../components/avatar">Avatar</a></li>
+              <li><a href="../components/alert">Alert</a></li>
+              <li><a href="../components/badge">Badge</a></li>
+              <li><a href="../components/button">Button</a></li>
+              <li><a href="../components/card">Card</a></li>
+              <li><a href="../components/image">Image</a></li>
+              <li><a href="../components/input">Input</a></li>
```
Comment (OmKakatkar) [inline color/index.html:83]: Broken Links, HTML file missing in URL

**Review by OmKakatkar — CHANGES_REQUESTED** [review]
Please have a look into all the HTML files, I may have missed some links.
Just adding the index.html to most of the broken links should solve the problem

File: color/index.html (hunk 12 above)
Comment (shrey27) [reply color/index.html:83]: Yeah links are still not connected properly. I will fix them in next update.

Comment (OmKakatkar) [reply color/index.html:83]: That's okay

**Pull request merged**